docs/howto/create-a-plugin.md

```diff
@@ -0,0 +1,24 @@
+# Create a Plugin
+After you [create a Workspace](./create-a-workspace.md), you can add a Plugin. To do so, use the create-ayx-plugin command. Reply to the prompts and then you will have the template code for your SDK Plugin. For this tool, we use the Input tool type.
```

Comment:
TW review: wrap `create-ayx-plugin` in code style (same as  `sdk-workspace-init` command in Create a Workspace page)